nevergrad/optimization/base.py

```diff
@@ -461,6 +464,14 @@ def minimize(
                     sleeper.stop_timer()
                 while self._finished_jobs:
                     x, job = self._finished_jobs[0]
+
+                    if max_time >= 0:
```

Comment:
I would rather this was not included in minimize , a callback added in the xps would be much better

Reply:
Callbacks: https://facebookresearch.github.io/nevergrad/optimizers_ref.html#callbacks

The reason is that this would avoid changing the API, especially since eventually we need to add some kind of time out, but probably not this way